ec2/sc-ec2-windows-jumpcloud.yaml

```diff
@@ -0,0 +1,323 @@
+            01_install_ms_vc:
+              command: !Join
+                - ''
+                - - 'Powershell.exe Set-ExecutionPolicy Bypass -Scope Process -Force;'
+                  - 'Powershell.exe C:\scripts\install-ms-vc.ps1 > C:\scripts\install-ms-vc.log'
+            02_install_jc_agent:
+              command: !Join
+                - ''
+                - - 'Powershell.exe Set-ExecutionPolicy Bypass -Scope Process -Force;'
+                  - 'Powershell.exe C:\scripts\install-jc-agent.ps1 '
+                  - ' -JcConnectKey '
+                  - Fn::Transform: {"Name": "SsmParam", "Parameters": {"Type": "SecureString", "Name": "/infra/JcConnectKey"}}
+                  - ' > C:\scripts\install-jc-agent.log'
+        config_jc:
+          files:
+            'c:\\scripts\\associate-jc-system.ps1':
+              source: "https://raw.githubusercontent.com/Sage-Bionetworks/infra-utils/v1.0.2/aws/associate-jc-system.ps1"
+              mode: "0664"
+          commands:
+            01_associate_jc_systems:
+              command: !Join
+                - ''
+                - - 'Powershell.exe Set-ExecutionPolicy Bypass -Scope Process -Force;'
+                  - 'Powershell.exe C:\scripts\associate-jc-system.ps1 '
+                  - ' -JcServiceApiKey '
+                  - Fn::Transform: {"Name": "SsmParam", "Parameters": {"Type": "SecureString", "Name": "/infra/JcServiceApiKey"}}
+                  - ' -JcSystemsGroupId '
+                  - Fn::Transform: {"Name": "SsmParam", "Parameters": {"Type": "SecureString", "Name": "/infra/JcSystemsGroupId"}}
+                  - ' -OwnerEmail '
+                  - !Ref OwnerEmail
+                  - ' > C:\scripts\associate-jc-system.log'
+    Properties:
+      ImageId: !Ref 'LatestAmiId'
+      InstanceType: !Ref 'WindowsInstanceType'
+      SubnetId: !ImportValue
+        'Fn::Sub': '${AWS::Region}-internalpoolvpc-PrivateSubnet1'
+      SecurityGroupIds:
+        - !Ref 'WindowsSecurityGroup'
+      KeyName: !Ref 'KeyPair'
+      IamInstanceProfile: !Ref 'PatchingInstanceProfile'
+      UserData:
+        Fn::Base64: !Sub |
+          <script>
+          cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetupJumpcloud
+          cfn-signal.exe -e %errorlevel% --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region}
+          </script>
+      Tags:
+        - Key: Name
+          Value: SC-EC2-RA-Windows-Instance
+        - Key: Description
+          Value: Service-Catalog-EC2-Reference-Architecture
+  WindowsPatchBaseline:
+    Type: AWS::SSM::PatchBaseline
+    Properties:
+      OperatingSystem: WINDOWS
+      ApprovalRules:
+        PatchRules:
+          - ApproveAfterDays: 0
+            ComplianceLevel: CRITICAL
+            PatchFilterGroup:
+              PatchFilters:
+                - Key: PRODUCT
+                  Values:
+                    - WindowsServer2016
```

Comment:
The deault `LatestAmiId` refers to a windows 2019 server so i'm thinking we should make this match?  You'll probably need to make sure that WindowsServer2019 is a valid product first though.  https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_DescribePatchProperties.html

Reply:
Changed in 143401c